README.md

```diff
@@ -1,12 +1,26 @@
 ## ALiAS: Amity Linux Assistance Sapience
-- The following repo contains the source code for the official website of Amity Linux Assistance Sapience. 
+
+ [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![GitHub license](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
+![GitHub issues](https://img.shields.io/github/issues-raw/asetalias/asetalias.github.io.svg) ![GitHub repo size](https://img.shields.io/github/repo-size/asetalias/asetalias.github.io.svg)
+![Twitter Follow](https://img.shields.io/twitter/follow/asetalias.svg?style=social)
+
+
+- The following repository contains the source code for the official website of Amity Linux Assistance Sapience. The largest open-source community of Amity University in Noida and now also in Lucknow. 
 - The website is hosted on gh-pages itself. 
-- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML Files.
+- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML files.
+
+<img src="https://user-images.githubusercontent.com/22801822/67155846-3d12c500-f334-11e9-9125-dbf32ea573a9.png" height=480>
```

Comment:
I can see the exit full-screen prompt.

Reply:
Oh, that's embarrassing. Let me add a new one.